packages/template-retail-react-app/app/pages/product-list/partials/refinements.jsx

```diff
@@ -38,9 +38,10 @@ const Refinements = ({filters, toggleFilter, selectedFilters, isLoading}) => {
 
     // Use saved state for accordions
     if (!isServer) {
+        // TODO: Change this to `useLocalStorage` hook when localStorage detection is more robust
```

Comment:
This is the only place in the retail react app that uses `localStorage` directly, so we want to make sure we handle it appropriately.